console/console-init/ui/src/modules/msg-and-iot/components/ProjectList/ProjectList.tsx

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright 2020, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+import React from "react";
+import {
+  Table,
+  TableVariant,
+  TableHeader,
+  TableBody,
+  SortByDirection,
+  IRowData,
+  sortable,
+  TableProps
+} from "@patternfly/react-table";
+import { Link } from "react-router-dom";
+import { FormatDistance } from "use-patternfly";
+import { StyleSheet, css } from "@patternfly/react-styles";
+import {
+  StatusTypes,
+  ProjectStatus,
+  ProjectError,
+  ProjectEntities,
+  ProjectTypePlan,
+  ProjectTypes
+} from "modules/msg-and-iot/utils";
+
+export const StyleForTable = StyleSheet.create({
```

Comment:
don't need to export styles, it should be for specific component only and i would like to suggest variable name as styles.

Reply:
The variable is imported in all list component like AddressList ConnectionList. that's why it is exported.